plugin/src/main/webapp/icons/LICENSE.txt

```diff
@@ -254,3 +254,8 @@ The Stylelint logo is licensed under CC BY 4.0.
 https://icon-icons.com/icon/Stylelint/131966
 
 ------------------------------------------------------------------------------
+
+The Aquasec logo is licensed under the GNU GENERAL PUBLIC LICENSE.
```

Comment:
Hmm, GPL is not compatible with MIT as far as I understand. On https://www.aquasec.com/brand/ I do not find the GPL reference (but there are only other tools listed). Is there a way to contact them if it's ok to reuse the icon here?

Reply:
I will try to contact them at some point. 
In the meantime Stylelint icon might need a review as well as it is using https://icon-icons.com/icon/Stylelint/131966.

I am curious if it would be better if we generated icons ourselves based on tool abbreviations?
That way it makes it easier to distinguish them on the UI rather than seeing the default icon.

Reply:
Looks like it has Creative Commons license.

![image](https://user-images.githubusercontent.com/27741935/182027898-9b97f4d0-0179-4ef1-b66f-f76f7ea2c525.png)
![image](https://user-images.githubusercontent.com/27741935/182027852-f94b1eb8-fecc-4eb4-9981-f0544e320397.png)